### ISSUE TITLE
target representation using one hot encoding instead of class index

### ISSUE THREAD

**Issue by aiwithshekhar** [meetshah1995/pytorch-semseg#229]
for this repo targets have shape (M,N) where all the pixels represent class index (0-21) and cross entropy loss has been used.
1) Is it possible to convert targets to one hot encoding with dimension (M,N,21) where all the pixels represent 0 or 1.
2) For this (M,N,21) representation of targets should we use the same loss or we should switch to dice loss?
